install/preflight/guard.rb

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+abort() {
+  echo -e "\e[31mOmarchy is only designed for a fresh Arch install!\e[0m"
+  exit 1
+}
+
+# Must be an Arch distro
+[[ -f /etc/arch-release ]] || abort
+
+# Must not be an Arch derivative distro
+for marker in /etc/cachyos-release /etc/eos-release /etc/garuda-release /etc/manjaro-release; do
```

Comment:
Maybe for a non-fresh Arch or a derivative distro an option to "Do you with to proceed on your own risks?"

I am now happy I could install omarchy before this check. Lol. :sweat_smile:

Reply:
Yeah, that's fair. We should allow people to cut themselves. Just don't want them to come running when they do 😂